### DISCUSSION TITLE
Graphical Issues

### DISCUSSION THREAD

**Discussion by MauroMardones** [nmfs-stock-synthesis/stock-synthesis#506] [Ideas]
Is there a way for the `SSplotData` function to aggregate the environmental data like it does with the other sources of information in the model?

**Comment by iantaylor-NOAA** [comment]
@MauroMardones,
This is a good idea. Could you post a new issue on the r4ss GitHub issue page and I'll work on it soon? (I'm still in transit home from ICES now).

**Reply by MauroMardones** [reply]
Thanks @iantaylor-NOAA. (me flying also!) . i going to the Issues!
